go.mod

```diff
@@ -225,10 +225,10 @@ require (
 
 require (
 	cloud.google.com/go/kms v1.15.2 // @grafana/backend-platform
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0 // @grafana/backend-platform
-	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys v0.9.0 // @grafana/backend-platform
-	github.com/Azure/azure-storage-blob-go v0.15.0 // @grafana/backend-platform
-	github.com/Azure/go-autorest/autorest/adal v0.9.22 // @grafana/backend-platform
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.3.0 // @grafana/partner-datasources
```

Comment:
All four of these dependencies do not directly belong to us.

Enterprise:
- `azidentity` (although this is technically shared as the `grafana-azure-sdk-go` package depends on it also)
- `azkeys`
- `autorest`

Backend platform:
- `azure-storage-blob-go` (used for the image uploader)

Reply:
Sorry totally got caught up in other things. 

- `azure-storage-blob-go can` probably go to alerting, will do in a separate PR - removed from this one
- anything in `grafana-azure-sdk-go` is owned by your squad https://github.com/grafana/grafana-azure-sdk-go/blob/main/.github/CODEOWNERS If there are some subparts that are shared ownership, we can add multiple squads here or change them but I would need to know who it would be.

Reply:
@tolzhabayev I understand we own everything in `grafana-azure-sdk-go`, however the following have a more direct dependency in `pkg/extensions` based on a `go mod why` (see my suggestions inline):

- `github.com/Azure/go-autorest/autorest` - We do not use this and should not be set as the owners. Our SDK does not depend on this and it is out of support.
- `github.com/Azure/azure-sdk-for-go/sdk/azidentity` - We can co-own this with the owners of `pkg/extensions`
- `github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys` - We do not use this and should not be set as the owner.
- `github.com/Azure/go-autorest/autorest/adal` - This is not owned by us and we do not use it. Nor does the SDK depend on it. This module is also deprecated and should be replaced by `azidentity`.

Reply:
Ok after further discussions I will try to switch it to someone else, only the sdk is left. So good to merge? @aangelisc